inc/integrations/airtable-integration/airtable-integration.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace RemoteDataBlocks\Integrations;
+
+use RemoteDataBlocks\Config\AirtableDatasource;
+use RemoteDataBlocks\Logging\LoggerManager;
+use RemoteDataBlocks\REST\DatasourceCRUD;
+
+require_once __DIR__ . '/datasources/airtable-datasource.php';
+
+class AirtableIntegration {
+	public static function init(): void {
+		self::register_dynamic_data_source_blocks();
+	}
+
+	private static function register_dynamic_data_source_blocks(): void {
+		$data_sources = DatasourceCRUD::get_data_sources( REMOTE_DATA_BLOCKS_AIRTABLE_SERVICE );
+
+		foreach ( $data_sources as $config ) {
+			// Transform data to our experimental format, which is all array based
+			$config = array_map(
+				function ( $value ) {
+					return is_object( $value ) ? (array) $value : $value;
+				},
+				(array) $config
+			);
```

Comment:
this was a hack when i was bumbling about in that unmerged try, but, at this point (hopefully) unnecessary.

(though I still think we should array everywhere personally!)